src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -368,7 +368,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' Compilation level parse options.  Used when compiling synthetic embedded code such as My template
         ''' </summary>
         ''' <returns>The Parse Options Setting.</returns>
-        Friend ReadOnly Property ParseOptions As VisualBasicParseOptions
+        Public ReadOnly Property ParseOptions As VisualBasicParseOptions
```

Comment:
@gafter @jaredpar @AnthonyDGreen I made this property public to get this info from IDE side. this one is weird since it is exposed through constructor and WithParseOption method (setter), but there is no getter.